File: ClientApp/src/app/models/leave/leave-hr-view.model.ts
export class LeaveHrView {
    constructor(employeeId?: string, employeeName?: string, department?: string, remainingLeave?: string) {

        this.employeeId = employeeId;
        this.employeeName = employeeName;
        this.department = department;
        this.remainingLeave = remainingLeave;

    }
    public employeeId: string;
    public employeeName: string;
    public department: string;
    public remainingLeave: string;
}
export class EmployeeLeaveDetails {
    constructor(leaveType?: string, allottedLeaves?: string, takenLeaves?: string, remainingLeave?: string) {

        this.leaveType = leaveType;
        this.allottedLeaves = allottedLeaves;
        this.takenLeaves = takenLeaves;
        this.remainingLeave = remainingLeave;
    }
    public leaveType: string;
    public allottedLeaves: string;
    public takenLeaves: string;
    public remainingLeave: string;
}
export class LeaveHrViewModel {

    constructor(leaveEmployeeList: LeaveHrView[], totalCount: number) {

        this.leaveEmployeeList = new Array<LeaveHrView>();
        this.totalCount = totalCount;
    }

    public leaveEmployeeList: LeaveHrView[];
    public totalCount: number;
    public isConfigSet: boolean;
}

File: A4.Empower-Git/A4.Empower/A4.BAL/Leave/LeaveHrViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace A4.BAL
{
    public class LeaveHrViewModel
    {
        public LeaveHrViewModel()
        {
            LeaveEmployeeList = new List<LeaveHrModel>();
        }
        public bool IsConfigSet { get; set; }
        public List<LeaveHrModel> LeaveEmployeeList { get; set; }
        public int TotalCount { get; set; }
    }

    public class LeaveHrModel
    {


        public string EmployeeId { get; set; }

        public string EmployeeName { get; set; }

        public string Department { get; set; }

        public int RemainingLeave { get; set; }

        public string EmployeeLeaveDetailsId { get; set; }
    }

    public class EmployeeLeaveDetails
    {
        public string LeaveType { get; set; }

        public int AllottedLeaves { get; set; }

        public int TakenLeaves { get; set; }

        public int RemainingLeave { get; set; }
    }
}
